compiler/src/iree/compiler/DispatchCreation/SetEncoding.cpp

```diff
@@ -32,6 +32,7 @@ namespace mlir::iree_compiler::DispatchCreation {
 #include "iree/compiler/DispatchCreation/Passes.h.inc"
 
 using IREE::Encoding::EncodingAttr;
+using IREE::Encoding::PackedStorageAttr;
```

Comment:
We do not need this, right? Can you drop the change?

Reply:
Oh this one eluded my constant rebasings. Thanks! Done.